2_process.R

```diff
@@ -105,6 +105,14 @@ p2_targets_list <- list(
     }
   ),
 
+  tar_target(
+    p2_rdsalt_per_catchment_allyrs,
+    Reduce(function(...) merge(..., by = c("POI_ID", "hru_id", "hru_segment_orig",
```

Comment:
Realizing after the fact that `purrr:Reduce()` might be a more tidier alternative here.

Reply:
Is there a preference for one or the other?

Reply:
I do not have a preference. They seem almost identical